packages/docusaurus-theme-search-algolia/src/theme/SearchBar/index.tsx

```diff
@@ -221,6 +222,149 @@ function DocSearch({
     description: 'The ARIA label and placeholder for search button',
   });
 
+  const translatedModal: DocSearchTranslations = {
```

Comment:
Can we extract all the translations to a separate file, or at least outside the component?

Reply:
agree 👍 we'll want to reduce the size of this component, and probably later also use some of those translations on the search page (quite old/legacy, but we'll want to refactor/upgrade someday)

Reply:
Um, it just means copying everything to another file like `src/theme/searchTranslations.ts`, and then importing it as `import translations from "@theme/searchTranslations"`...